requirements/frozen_docs.py3.txt

```diff
@@ -35,8 +35,10 @@ fauxfactory==3.0.2
 flake8==3.5.0
 funcsigs==1.0.2
 future==0.16.0
+gevent==1.3.5
```

Comment:
These are mocked in the docs build, but freeze_all is going to keep adding them, so FINE lets let it try to do a docs build with them. :)